src/server/pmix_server_get.c

```diff
@@ -457,7 +466,8 @@ static pmix_status_t _satisfy_request(pmix_nspace_t *nptr, pmix_rank_t rank,
             PMIX_DESTRUCT(&pbkt);
             cbfunc(rc, NULL, 0, cbdata, NULL, NULL);
             return rc;
-        }
+        }    
+	if (rank == PMIX_RANK_WILDCARD) found++;
```

Comment:
Minor nit: coding standards require use of braces around every if clause as it helps when running a debugger